### PR TITLE
Blog

### PR DESCRIPTION
**Pull request by rich-howell**
https://docusaurus.io/docs/blog
https://deploy-preview-196--actualbudget-website.netlify.app/

### CODE REVIEW

**Comment by rich-howell** [comment]
Work in progress, working on the blog posts.

**Comment by j-f1** [comment]
Would you be interested in auto-generating the release blog posts from the release notes page at build time? Or do you think we should get rid of the release notes page entirely in favor of just linking to https://deploy-preview-196--zingy-bonbon-b1b286.netlify.app/blog/tags/release?

**Comment by rich-howell** [comment]
All the core work is done for the blog section of this I think, ready for feedback. 

Would appreciate reviews if you would like to make changes so I can see what we are all thinking :)

File: blog/2022-10-25-release-22.10.25.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+description: New release of Actual.
+date: 2022-10-25T10:00
+slug: release-22.10.25
+authors: rich-howell
+image: /img/blog/22.10.25.png
+tags: [announcement, release]
+hide_table_of_contents: false
+---
+
+![](/img/blog/22.10.25.png)
```
Comment (j-f1) [inline blog/2022-10-25-release-22.10.25.md:12]: How are these images generated? Can you publish the sources? (or is there a way we could auto-generate them)? Also, could you export them as an .svg or at least at a higher resolution if possible? They look a little blurry on my 2x screen.

File: blog/2022-10-25-release-22.10.25.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+- [#64](https://github.com/actualbudget/actual-server/pull/64) build: add node GC argument to fly template — thanks [PartyLich]
+- [#65](https://github.com/actualbudget/actual-server/pull/65) build: add tini subreaper arg to fly template — thanks [PartyLich]
+- [#70](https://github.com/actualbudget/actual-server/pull/70) Update Express to version 4.17 — thanks [rich-howell]
+- [#72](https://github.com/actualbudget/actual-server/pull/72) Fix/download only necessary files — thanks [PartyLich]
+- [#75](https://github.com/actualbudget/actual-server/pull/75) Switch syncing to simple sync method — thanks [jlongster]
+- [#78](https://github.com/actualbudget/actual-server/pull/78) Respect configuration for user-files and don't init the app — thanks [jlongster]
+- [#81](https://github.com/actualbudget/actual-server/pull/81) Store user files as blobs instead of unzipping them — thanks [jlongster]
+- [#82](https://github.com/actualbudget/actual-server/pull/82) Build docker image on push to master or tag — thanks [trevdor]
+
+[7brend7]: https://github.com/7brend7
```
Comment (j-f1) [inline blog/2022-10-25-release-22.10.25.md:130]: Will open a PR to make this less repetitive.

File: blog/welcome.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+title: Welcome to Actual Budget Open Source
+description: The new home for the Open Source project.
+slug: welcome-actualbudget-os
+authors: rich-howell
+tags: [announcement, coreproject]
+image: https://i.imgur.com/mErPwqL.png
```
Comment (j-f1) [inline blog/welcome.md:7]: Can this be removed or copied into the repo?

File: blog/2023-06-01-release-23.6.0.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+slug: release-23.6.0
+authors: rich-howell
+tags: [announcement, release]
+image: /img/blog/23.6.0.png
+hide_table_of_contents: false
+---
+
+![](/img/blog/23.6.0.png)
+
+**Docker tag: 23.6.0**
```
[the remark this answers is not in the document]
Comment (rich-howell) [reply blog/2023-06-01-release-23.6.0.md:14]: Fixed this

File: blog/2022-10-25-release-22.10.25.md (hunk 1 above)
Comment (rich-howell) [reply blog/2022-10-25-release-22.10.25.md:12]: I created them, I will pull them down, no way of auto generating them.

File: blog/2022-10-25-release-22.10.25.md [hunk 5]
```diff
@@ -0,0 +1,193 @@
+---
+title: Release 22.10.25
+description: New release of Actual.
```
[the remark this answers is not in the document]
Comment (rich-howell) [reply blog/2022-10-25-release-22.10.25.md:3]: There isn't always a first line to use.

File: blog/2023-06-01-release-23.6.0.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+
+<!--truncate-->
+
+### Actual
```
[the remark this answers is not in the document]
Comment (rich-howell) [reply blog/2023-06-01-release-23.6.0.md]: Done

File: blog/2023-06-01-release-23.6.0.md [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+tags: [announcement, release]
+hide_table_of_contents: false
+---
+
+The release has the following notable improvements:
+
+Ability to hide categories from budget page
+Custom filters in reports
+Experimental Nordigen configuration via the UI (simplified onboarding)
+Experimental goal template improvements
```
[the remark this answers is not in the document]
Comment (rich-howell) [reply blog/2023-06-01-release-23.6.0.md]: They are - your just too quick

**Pull request merged**